src/parser/open-api/oneOfResolver.js

```diff
@@ -0,0 +1,145 @@
+import cloneDeep from 'lodash/cloneDeep'
+import get from 'lodash/get'
+import update from 'lodash/update'
+import toPath from 'lodash/toPath'
+
+/**
+ * Recursively determine if the given object has any `oneOf` properties.
+ * Returns an array with the paths to those properties (eg. `properties.oneOf`).
+ * If no `oneOf` keys were found, an empty array is returned.
+ *
+ * @param {object} obj
+ * @return {array}
+ */
+function getOneOfPaths (obj) {
```

Comment:
Nice!
If you want, you could rename `k` to `key`. It would be more descriptive.